### PR TITLE
Add Rob's API docs (#14) to the docs directory

### PR DESCRIPTION
**Pull request by openjck**

### CODE REVIEW

**Comment by robhudson** [comment]
Looks good. On Monday I'll double check that the keys match the actual output from the API. I think I pulled that sample from the sample data set we got, not the API output.

File: docs/api.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+The API outputs data in the following format:
+
+```
+[
+  {
+    "numObs": 5321560,
+    "type": "category",
+    "metric": "architecture",
+    "data": [
```
Comment (robhudson) [inline docs/api.md]: Instead of `"data"` we call this `"points"` in the actual API.

**Comment by openjck** [comment]
Updated! :smile:

**Comment by robhudson** [comment]
We can merge this as-is. But ultimately I think adding more details about what the API URL is and adding the metrics API and per-metric distributions API would be nice.

**Comment by openjck** [comment]
Updated. Is this what you had in mind?

**Comment by robhudson** [comment]
Yep! Looks great. Thanks!

**Pull request merged**